docker/docker-compose-loofah-noAPI.yml

```diff
@@ -7,6 +7,15 @@ services:
     ports:
     - 27017:27017
 
+  loofah-db-init:
```

Comment:
Is this docker-compose file intended for running up the frontend only? If so, why does it need the database and db-init?

Reply:
I _think_ @vijamesDD uses this to run the API in intellij so that she can see API changes real quick